common/djangoapps/student/views.py

```diff
@@ -632,11 +637,8 @@ def dashboard(request):
         # Re-alphabetize language options
         language_options.sort()
 
-    # TODO: remove circular dependency on openedx from common
-    from openedx.core.djangoapps.user_api.models import UserPreference
-
-    # try to get the prefered language for the user
-    cur_pref_lang_code = UserPreference.get_preference(request.user, LANGUAGE_KEY)
+    # try to get the preferred language for the user
+    cur_pref_lang_code = preferences_api.get_user_preference(request.user, LANGUAGE_KEY)
```

Comment:
Could we rename this since we're touching it anyway? Something like ``preferred_language``.